tools/docker-compose-cluster.yml

```diff
@@ -66,12 +65,12 @@ services:
       RABBITMQ_PASS: guest
       RABBITMQ_VHOST: /
       SDB_HOST: 0.0.0.0
-      SDB_PORT: 7899
```

Comment:
This conflicts w/ the new default port range on `awx_2` and needs to be bumped up.